src/main/target/KISSFC/target.h

```diff
@@ -84,7 +84,7 @@
 #define UART3_RX_PIN            PB11 // PB11 (AF7)
 
 #ifdef KISSCC
-#define SOFTSERIAL1_TX_PIN      PA13
+#define SOFTSERIAL1_TX_PIN      PA13 // TLM
```

Comment:
@robertlacroix: Can you simplify this please?

Reply:
Yeah, sorry busy the last couple of days. I’ll send an updated change soon

Reply:
@mikeller done